### DISCUSSION TITLE
GeoParquet index via naming scheme ?

### DISCUSSION THREAD

**Discussion by ymoisan** [OvertureMaps/data#157] [Discussion]
Hello.  I'm trying to download one geoparquet file for my home town : Sherbrooke, Québec, Canada (45° 24′ N, 71° 54′ W).  I'll set up DuckDB in my conda env if I need to, but is there not some documentation regarding the part naming scheme that has a link to coordinate bboxes that I can use to find the one file I need ?

My final use case is to load the geoparquet file in QGIS next to other buildings data in GeoPackage format and compare.

Thank you for this great project.

```
s3.ls('s3://overturemaps-us-west-2/release/2024-04-16-beta.0/theme=buildings/type=building')

Last executed at 2024-05-14 14:58:08 in 4ms

['overturemaps-us-west-2/release/2024-04-16-beta.0/theme=buildings/type=building/part-00000-530a22ea-6b33-45e9-961b-567f661900b0-c000.zstd.parquet',
 'overturemaps-us-west-2/release/2024-04-16-beta.0/theme=buildings/type=building/part-00001-530a22ea-6b33-45e9-961b-567f661900b0-c000.zstd.parquet',
 'overturemaps-us-west-2/release/2024-04-16-beta.0/theme=buildings/type=building/part-00002-530a22ea-6b33-45e9-961b-567f661900b0-c000.zstd.parquet',
 'overturemaps-us-west-2/release/2024-04-16-beta.0/theme=buildings/type=building/part-00003-530a22ea-6b33-45e9-961b-567f661900b0-c000.zstd.parquet',
 'overturemaps-us-west-2/release/2024-04-16-beta.0/theme=buildings/type=building/part-00004-530a22ea-6b33-45e9-961b-567f661900b0-c000.zstd.parquet',
...
```

**Comment by mtravis** [comment]
See answer to another issue from @jenningsanderson https://github.com/OvertureMaps/data/issues/127#issuecomment-2073647662

I would install DuckDB and have a play. It's a great tool and works really well with Overture data.

**Reply by jenningsanderson** [reply]
Yes, the linked issue there includes a query to identify which parquet file contains data for a particular area of interest — but we highly recommend using a tool like DuckDB to query Overture data because it will handle that part for you.

**Comment by ymoisan** [comment]
I ended up using `overturemaps` directly in a notebook as per [this notebook](https://developmentseed.org/lonboard/latest/examples/overture-maps/).  I like that I can get data for an arbitrary bbox.  The pyarrow table we get from `overturemaps.record_batch_reader` can be turned into a pandas df then saved as parquet.
